brigade.js

```diff
@@ -28,3 +45,7 @@ events.on("check_suite:rerequested", runSuite);
 
 events.on("issue_comment:created", (e, p) => Check.handleIssueComment(e, p, runSuite));
 events.on("issue_comment:edited", (e, p) => Check.handleIssueComment(e, p, runSuite));
+
+events.on("push", (e, p) => {
```

Comment:
So on `check_suite:requested` we build the Node library, and on a `push` we publish the images?
Isn't the `push` event triggered even on branch pushes? If so, we end up executing both on all branch pushes as well.

Reply:
That's correct.  Note that the `check_suite` events will be triggered on all Pull Requests.  Currently, I figured we wouldn't want to build and publish these images for every pull request.  However, we could do so.  Or, perhaps a good compromise would be to build the images for all checks (PRs) but only push/publish images in response to the `push` event?